src/scheduler/ExecutableIntervalSchedule.ts

```diff
@@ -0,0 +1,65 @@
+import humanInterval from 'human-interval';
+import { max } from 'lodash';
+import { DateTime } from 'luxon';
+import { IntervalSchedule } from '../job/MomoJob';
+import { momoError } from '../logging/error/MomoError';
+import { TimeoutHandle, setSafeIntervalWithDelay } from '../timeout/setSafeIntervalWithDelay';
+import { Logger } from '../logging/Logger';
+import { ExecutableSchedule, NextExecutionTime } from './ExecutableSchedule';
+import { ExecutionInfo } from '../job/ExecutionInfo';
+
+export class ExecutableIntervalSchedule implements ExecutableSchedule<IntervalSchedule> {
+  private readonly interval: string;
+  private readonly firstRunAfter: number;
+  private timeoutHandle?: TimeoutHandle;
+
+  constructor({ interval, firstRunAfter }: IntervalSchedule) {
+    this.interval = interval;
+    this.firstRunAfter = firstRunAfter;
+  }
+
+  toObject(): IntervalSchedule {
+    return { interval: this.interval, firstRunAfter: this.firstRunAfter };
+  }
+
+  execute(callback: () => Promise<void>, logger: Logger, executionInfo?: ExecutionInfo): NextExecutionTime {
+    const interval = this.parse();
+    const delay = this.calculateDelay(interval, executionInfo);
+
+    this.timeoutHandle = setSafeIntervalWithDelay(callback, interval, delay, logger, 'Concurrent execution failed');
+
+    return { date: DateTime.fromMillis(DateTime.now().toMillis() + delay) };
+  }
+
+  isStarted(): boolean {
+    return this.timeoutHandle !== undefined;
+  }
+
+  stop(): void {
+    if (this.timeoutHandle) {
```

Comment:
looks like we're a bit inconsistent, we should decide if we want to write `if(foo !== undefined)` vs `if(foo)`

I noticed it in the Executable***Schedule classes, but probably whole project should be checked again ... I guess we could also configure the linter to force us to explicitely write `=== undefined` if that's the option we agree to use ....

Reply:
Hm... actually it seems we've explicitly set the rule to `allowNullableObject`. So I'd rather undo the `=== undefined` ones.

Reply:
Ok, now the rules are more strict. I've fixed the undefined checks I found.